server/hooks.go

```diff
@@ -68,10 +107,12 @@ func (p *Plugin) UserHasJoinedChannel(c *plugin.Context, channelMember *model.Ch
 	// We send a DM and an opportunistic ephemeral message to the channel. See
 	// the discussion at the link below for more details:
 	// https://github.com/mattermost/mattermost-plugin-welcomebot/pull/31#issuecomment-611691023
+	channelAddMessage := "# Welcome to the " + channelInfo.DisplayName + " channel. \n\n"
```

Comment:
See above. Why is this behavior change added here?

Reply:
This change I believe was added to give some kind of header to the welcome bot messages. However, if this ultimately feels like we are stripping freedoms to the welcome message, it can be thrown out.

Reply:
Yes, please throw it out.